### DISCUSSION TITLE
Docs idea: GeoVista through the scientific journey

### DISCUSSION THREAD

**Discussion by trexfeathers** [bjlittle/geovista#1204] [Ideas]
## 📚 Documentation

Early in GeoVista's life there were worries that it might just be a shiny toy and be unable to play a part in 'real science'. I don't worry about this anymore; GeoVista is proving itself very capable and it would be good to demonstrate this to the world in a documentation page - GeoVista is able to help at all stages of the scientific method:

### 👁 Observation and question

Perhaps GeoVista's strongest area. It enables people to quickly interact with their data in a highly intuitive way, possibly moreso than any other package. With panning, zooming, animation, shape rendering, relief-warping... there must be countless examples we could draw on.

### 🧪 Hypothesis and experiment

GeoVista's capabilities enable workflows that test hypotheses which would otherwise be un-testable. This could be the visualisation of models/observations that couldn't be rendered in any other way, or the unique regional extraction tools that we're just getting started with ([`geodesic.py`](https://github.com/bjlittle/geovista/blob/b84918611f579924871c122e5f2a42c0b5323919/src/geovista/geodesic.py), [geovista-slam](https://github.com/bjlittle/geovista-slam)).

### 💬 Analysis and reporting conclusions

Again, GeoVista can render things that cannot be rendered by other means. We know that GeoVista plots have been used within publications, so they must have met the standards of peer review. Perhaps we could link to some of these publications?

I would like to hear more about GeoVista's usefulness in these final stages of the scientific method.

**Comment by bjlittle** [comment]
@trexfeathers Do you think this is more suited to be a GH Discussion rather than an issue?

**Comment by trexfeathers** [comment]
> @trexfeathers Do you think this is more suited to be a GH Discussion rather than an issue?

Your call 👍. I wasn't necessarily calling for comments so much as posting an idea for the future, but if you see some space for discussion then that's cool.
